components/generic/RelatedChip.vue

```diff
@@ -9,7 +9,7 @@
   >
     <b-img
       v-if="img"
-      :src="img"
+      :src="`${img}&size=w200`"
       :alt="title"
```

Comment:
While not from this branch, let's change this to `alt=""`, because the title text that follows is identical, so this is worse for accessibility.

Reply:
yep that's a good point about accessibility